localisation_synced/hfe_empire_formats_l_default.yml

```diff
@@ -2,4 +2,6 @@
  human_fallen_emires_test_string_0: "<human_fallen_emires_test_string_random> 1"
  human_fallen_emires_test_string_1: "[This.GetName] 1"
  human_fallen_emires_test_string_2: "text 1"
- hfe_init.50.1.3.1_test: "[This.GetName]"
\ No newline at end of file
+ hfe_init.50.1.3.1_test: "[This.GetName]"
+ human_fallen_empires_names_pirate_0_0:0 "Pirate"
```

Comment:
Why you need this strings

Reply:
The CW Tools were spamming me about this in multiple lines. I had no idea why, but this fixed them.